provider/cluster/client.go

```diff
@@ -1,12 +1,17 @@
 package cluster
 
-import "github.com/ovrclk/akash/types"
+import (
+	"github.com/ovrclk/akash/types"
+	"k8s.io/api/apps/v1"
+)
 
 type Client interface {
 	Deploy(types.LeaseID, *types.ManifestGroup) error
 	Teardown(types.LeaseID) error
 
 	Deployments() ([]Deployment, error)
+	KubeDeployments(types.LeaseID) (*v1.DeploymentList, error)
+	KubeDeployment(types.LeaseID, string) (*v1.Deployment, error)
```

Comment:
I'd prefer to keep this ignorant of kube -- make a simplified, unified status struct -- but for now it's ok.